test/types/index.ts

```diff
@@ -269,6 +269,14 @@ server.setSchemaCompiler(function (schema: object) {
 
 server.addSchema({})
 
+server.addContentTypeParser('foo/bar', {}, (req, done) => {
+  done!(null, {})
+})
+
+server.addContentTypeParser('foo/bar', {}, async (req: http2.Http2ServerRequest) => [])
+
+server.hasContentTypeParser('foo/bar')
```

Comment:
can you add a check for testing that the returned value is a boolean? (if there's a way to do that)

Reply:
Done

Reply:
@ShogunPanda I'm not able to see that test. Could you point it to me?

Reply:
@allevo Sorry, I forgot to push the changes. Now it's there.